config/default.go

```diff
@@ -245,6 +244,9 @@ RequireValidatorCall = false
 		MinConnectTimeout = "5s"
 		RequestTimeout = "30s"
 		UseTLS = false
+	[AggSender.StorageRetainCertificatesPolicy]
+		RetainNCertificates = 0 # 0 means keep all certificates
```

Comment:
I find it slightly more readable, WDYT?
```suggestion
		RetainCertificatesCount = 0 # 0 means keep all certificates
```

Reply:
done: 9d03f6d5ce83086a5f7c2c24a934bf1b42a59486